### PR TITLE
Added Alt, Width and Height properties to CustomCollectionImage.

### PR DESCRIPTION
**Pull request by david-mclean**
Added, Alt, Width and Height to CustomCollectionImage inline with the shopify documentation here: https://help.shopify.com/en/api/reference/products/customcollection

### CODE REVIEW

File: ShopifySharp/Entities/CustomCollectionImage.cs [hunk 1]
```diff
@@ -20,10 +20,28 @@ public class CustomCollectionImage
...
         /// The date the image was created.
         /// </summary>
         [JsonProperty("created_at")]
         public DateTimeOffset? Created_At { get; set; }
+
+        /// <summary>
+        /// Width of the image in pixels.
+        /// </summary>
+        [JsonProperty("width")]
+        public int Width { get; set; }
```
Comment (nozzlegear) [inline ShopifySharp/Entities/CustomCollectionImage.cs]: Could you change this to `int?` to match the rest of the package?

File: ShopifySharp/Entities/CustomCollectionImage.cs [hunk 2]
```diff
@@ -20,10 +20,28 @@ public class CustomCollectionImage
...
+
+        /// <summary>
+        /// Height of the image in pixels.
+        /// </summary>
+        [JsonProperty("height")]
+        public int Height { get; set; }
```
Comment (nozzlegear) [inline ShopifySharp/Entities/CustomCollectionImage.cs]: This one should also be `int?`

**Comment by david-mclean** [comment]
No problem, changed.

**Comment by nozzlegear** [comment]
Published in 4.21.7, thank you!

**Pull request merged**